exporter/containerimage/writer.go

```diff
@@ -153,6 +157,25 @@ func (ic *ImageWriter) Commit(ctx context.Context, inp exporter.Source, sessionI
 		idx.Manifests = append(idx.Manifests, *desc)
 
 		labels[fmt.Sprintf("containerd.io/gc.ref.content.%d", i)] = desc.Digest.String()
+
+		if attestations, ok := inp.Attestations[p.ID]; ok {
+			inTotos, err := ic.exportAttestations(ctx, session.NewGroup(sessionID), desc, attestations...)
+			if err != nil {
+				return nil, err
+			}
+
+			desc, _, err := ic.commitAttestationsManifest(ctx, p, desc.Digest.String(), opts.OCITypes, attestations, inTotos)
+			if err != nil {
+				return nil, err
+			}
+			desc.Platform = &ocispecs.Platform{Architecture: "none", OS: "none"}
```

Comment:
Few thoughts here, maybe `undefined` or `unspecified` would be a little better then `none`?

Also, what are we thinkin of doing about platform-specific attesttions, should that be somehow internal to the the attestation bundle itself?

Reply:
Under the model we have, all attestations are platform specific - so each platform's attestations will be in it's own `none/none` manifest. Though if the attestation content is the same, we won't upload the data twice, since we'll have the same digest.

`none` is just a placeholder for now, I don't have strong opinions on what it should be.